force-app/main/default/staticresources/lmsvf/lmsvf.js

```diff
@@ -0,0 +1,149 @@
+"use strict";
+
+var lmsutil = window.lmsutil; 
+
+if (lmsutil === undefined) {
+    lmsutil = {};
+}
+
+lmsutil.lmsChannelSubscription; 
+
+// Function to generate li elements from contact remote object record
+lmsutil.constructContactListItem = function(item){
+    var nodeString = `
+        <li data-id="${item.get('Id')}">
+            <a href="#">
+                <div class="slds-grid slds-grid_vertical-align-center">
+                    <div>
+                        <img class="img-thumb" src="${item.get('Picture__c')}">
+                    </div>
+                    <div class="slds-p-around_small">
+                        ${item.get('Name')}
+                    </div>
+                </div>
+            </a
+        </li>
+    `;
+
+    return nodeString.trim(); 
```

Comment:
Can't you move the `` ` `` on the same line as the string start/end and get rid of that `trim()`?